cloud-watch-to-slack-testing-58006136-c30b-43d1-880c-7b36f860e7c8/deployment/index.js

```diff
@@ -134,23 +123,13 @@ function processEvent(event, callback) {
 
 
 exports.handler = (event, context, callback) => {
+    // Uncomment the folling line to see the event in the CloudWatch logs
+    //console.info("cloud-watch-to-slack-testing EVENT\n" + JSON.stringify(event, null, 2));
```

Comment:
You could probably just leave this uncommented.

Reply:
now uncommented